src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -229,7 +230,33 @@ const formConfig = {
           }
         },
         // pageFour: {},
-        // pageFive: {},
+        privateMedicalRecordsIntro: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-intro',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => _.get(`disabilities.${index}.view:privateMedicalRecords`, formData),
```

Comment:
Should these properties be in `[]`?

Reply:
They could be, sure, but it works just fine like this. :man_shrugging:
I went with a string template over array for the slight readability win.